server/build.gradle

```diff
@@ -58,6 +58,9 @@ dependencies {
   testImplementation 'org.mockito:mockito-core:5.1.1'
 
   testImplementation 'com.mockrunner:mockrunner-servlet:2.0.6'
+
+  // The Mockito Java mocking library
+  testImplementation "org.mockito:mockito-core:3.+"
```

Comment:
This would have been new when it was first a branch... but it's not going to write over the newer version here, is it?

Reply:
It won't _overwrite_ it, but it will conflict with the 5.1.1 definition on line 58. We definitely don't want these lines.